pom.xml

```diff
@@ -17,6 +17,7 @@
         but if multiple dependencies are linked, use a property for single
         dependency source) -->
         <cucumber.version>4.3.1</cucumber.version>
+        <jacoco.version>0.8.5</jacoco.version>
```

Comment:
jacoco.version is defined twice.  Here on line 20 and later on line 28.  This line should be removed.